lib/config.js

```diff
@@ -16,6 +16,19 @@ const getNPMConfig = (path) => {
     process.env[package_prefix + key]
 }
 
+const parseExtraInputs = (inputs, accumulator, callback) => {
+  for (let input of inputs) {
+    let separatorIndex = input.indexOf(':')
```

Comment:
`split`?

Reply:
There could be colons in the value as well. We only want to extract up to the first colon.